schema/fields/date.js

```diff
@@ -7,16 +8,19 @@ export default {
     format: {
       type: GraphQLString,
     },
+    timezone: {
+      type: GraphQLString,
+    },
     convert_to_utc: {
       type: GraphQLBoolean,
-      defaultValue: true,
+      deprecationReason: 'Use timezone instead',
```

Comment:
Should this just be removed completely? Doesn't look like it gets used at all anymore.